issue1〜3の修正

NaotoToriizuka opened the pull request.

Commit:
Issue1

Commit:
Issue2

Commit:
Issue3

Katsuno0814 commented on app/controllers/calendars_controller.rb:
```diff
@@ -18,8 +18,8 @@ def plan_params
-  def getWeek
-    wdays = ['(日)','(月)','(火)','(水)','(木)','(金)','(土)']
+  def get_week
+    week_days = ['(日)','(月)','(火)','(水)','(木)','(金)','(土)']
```
こちらは今回のissueでは無い部分ですのでwdaysに戻していただいて問題ありません！

NaotoToriizuka replied:
修正しました。

Katsuno0814 reviewed (commented):
レビュー依頼ありがとうございます！
コメントしましたので、ご確認をお願いいたします！

Commit:
issue1〜3の修正

week_days = ['(日)','(月)','(火)','(水)','(木)','(金)','(土)']をwdays = ['(日)','(月)','(火)','(水)','(木)','(金)','(土)']に戻す。

ky3090 commented:
レビュー依頼ありがとうございます！
無事全てのissueを解決することができましたね！
LGTMとさせていただきます！お疲れ様でした！

こちらの画面よりマージしていただき、ローカルリポジトリでマスターブランチに戻りgit pull後
新たなブランチを作成して次のissueに取り組みましょう！応援しております！
https://master.tech-camp.in/curriculums/4681
